content/en/api/tags/tags_delete.md

```diff
@@ -9,8 +9,5 @@ external_redirect: /api/#remove-host-tags
 This endpoint allows you to remove all tags in an integration source for a single host.
 
 ##### ARGUMENTS
```

Comment:
Not sure how you are suppose to remove all tags from a source unless you specify the source. Either the argument is needed or the description needs to change.

Reply:
The description is wrong, it should remove tags associated to a host and that's it.